platform/android/export/export_plugin.cpp

```diff
@@ -1037,9 +1037,14 @@ void EditorExportPlatformAndroid::_write_tmp_manifest(const Ref<EditorExportPres
 	store_string_at_path(manifest_path, manifest_text);
 }
 
+bool EditorExportPlatformAndroid::_should_be_transparent(const Ref<EditorExportPreset> &p_preset) const {
+	return (bool)get_project_setting(p_preset, "display/window/per_pixel_transparency/allowed") &&
+			(bool)get_project_setting(p_preset, "display/window/size/transparent") &&
+			(bool)get_project_setting(p_preset, "rendering/viewport/transparent_background");
```

Comment:
`rendering/viewport/transparent_background` check is probably excessive, on desktop platforms setting `display/window/size/transparent`  (or `transparent` flag for the `Window`) will automatically set viewport background to transparent regardless of this setting.